test/application_system_test_case.rb

```diff
@@ -6,6 +6,15 @@
 
 ActionDispatch::SystemTesting::Server.silence_puma = true
 
+if Rails::VERSION::MAJOR < 7
+  # Necessary so that Capybara::Selenium::DeprecationSuppressor is prepended in
+  # Selenium::WebDriver::Logger before it is instantiated in
+  # Selenium::WebDriver.logger to prevent an uninitialized instance variable
+  # warning.
```

Comment:
Is it the `@suppress_for_capybara` ivar that's uninitialized if we don't load selenium & prepend `Capybara::Selenium::DeprecationSuppressor`? 🤔  I don't see any warnings, even if I comment out the `load_selenium` line, and wondering what I'm missing 😄

Reply:
Ah, version thing, got it 👍  Thanks!